pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/store/SingleFileIndexDirectory.java

```diff
@@ -387,4 +401,53 @@ public boolean isIndexRemovalSupported() {
   public String toString() {
     return _segmentDirectory.toString() + "/" + _indexFile.toString();
   }
+
+  @VisibleForTesting
+  static List<IndexEntry> copyIndicesTo(File indexFile, Map<IndexKey, IndexEntry> toBeRetained, File tmpIdxFile)
```

Comment:
Rename the parameters
```suggestion
  static List<IndexEntry> copyIndices(File srcFile, File destFile, Map<IndexKey, IndexEntry> indicesToCopy)
```

Reply:
Also add some javadoc